src/YoTennis/Models/State.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace YoTennis.Models
+{
+    public class State
+    {
+        public PlayerScore ScoreOnSets { get; set; }
+        public List<Set> ScoreInSets { get; set; }
+        public Game ScoreInGame { get; set; }
+
+        public DateTime GameTime { get; set; }
+        public Player PlayerOnLeft { get; set; }
+        public Player PlayerServes { get; set; }
+        public ServePositionOnTheCenterLine ServePositionOnTheCenterLine { get; set; }
+        public bool SecondServe { get; set; }
+
+        public DateTime MatchDate { get; set; }
+        public MatchSettings MatchSettings { get; set; }
+        public string FirstPlayer { get; set; }
+        public string SecondPlayer { get; set; }
+
+        public MatchState MatchState { get; set; }
+
+        public ServeSpeed ServeSpeed { get; set; }
```

Comment:
Why it's here in the state? Because service speed is on every serve, but this one is in the state. So is it last or average?